From ced6d3acc016b59d77cdc83a1d4054c6c953a783 Mon Sep 17 00:00:00 2001
From: Pascal Paillet <p.paillet@foss.st.com>
Date: Tue, 3 Sep 2024 18:03:39 +0200
Subject: [PATCH 1130/1695] dts: stm32mp15: remove useless gpio regulator
 property

Remove gpio-states property from the STM32MP15 device-trees
because it is not parsed by the gpio regulator driver.

Upstream-Status: Inappropriate [ST downstream https://github.com/STMicroelectronics/optee_os.git eeebd8ef3af13a4881ad02da64e83f342a3e06cf dts: stm32mp15: remove useless gpio regulator property]
Change-Id: I277b9c6e78863d3e5474800554f74d2fa9045b46
Signed-off-by: Pascal Paillet <p.paillet@foss.st.com>
Reviewed-on: https://gerrit.st.com/c/mpu/oe/optee/optee_os/+/401838
ACI: CIBUILD <MDG-smet-aci-builds@list.st.com>
Domain-Review: Patrick DELAUNAY <patrick.delaunay@foss.st.com>
Tested-by: Pascal PAILLET <p.paillet@st.com>
Reviewed-by: Thomas BOURGOIN <thomas.bourgoin@foss.st.com>
ACI: CITOOLS <MDG-smet-aci-reviews@list.st.com>
Reviewed-by: Pascal PAILLET <p.paillet@st.com>
---
 core/arch/arm/dts/stm32mp157c-ed1.dts              | 1 -
 core/arch/arm/dts/stm32mp15xx-dhcor-avenger96.dtsi | 1 -
 2 files changed, 2 deletions(-)

diff --git a/core/arch/arm/dts/stm32mp157c-ed1.dts b/core/arch/arm/dts/stm32mp157c-ed1.dts
index 5b844684f..68362eb10 100644
--- a/core/arch/arm/dts/stm32mp157c-ed1.dts
+++ b/core/arch/arm/dts/stm32mp157c-ed1.dts
@@ -86,7 +86,6 @@
 		regulator-always-on;
 
 		gpios = <&gpiof 14 GPIO_ACTIVE_HIGH>;
-		gpios-states = <0>;
 		states = <1800000 0x1>,
 			 <2900000 0x0>;
 	};
diff --git a/core/arch/arm/dts/stm32mp15xx-dhcor-avenger96.dtsi b/core/arch/arm/dts/stm32mp15xx-dhcor-avenger96.dtsi
index 76c54b006..743d440d6 100644
--- a/core/arch/arm/dts/stm32mp15xx-dhcor-avenger96.dtsi
+++ b/core/arch/arm/dts/stm32mp15xx-dhcor-avenger96.dtsi
@@ -81,7 +81,6 @@
 		regulator-always-on;
 
 		gpios = <&gpioi 5 GPIO_ACTIVE_HIGH>;
-		gpios-states = <0>;
 		states = <1800000 0x1>,
 			 <2900000 0x0>;
 	};
-- 
2.47.2

